public/catalog/v1.0.0/openapi.json

```diff
@@ -979,6 +979,46 @@
           }
         }
       }
+    },
+    "/portfolio_items/{portfolio_item_id}/icon": {
+      "get": {
+        "tags": [
+          "users",
```

Comment:
@syncrou @lindgrenj6 Use the new tags Portfolio

Reply:
rebased on master, updated to use new tag.